### PR TITLE
{bio}[foss/2021b] DIALOGUE v1.0-20230228

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires:
* #17695

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/db7d3a71cc0894b6cd4d8aadc04eb720 for a full test report.

File: easybuild/easyconfigs/d/DIALOGUE/DIALOGUE-1.0-20230228-foss-2021b-R-4.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'RPackage'
+
+name = 'DIALOGUE'
+local_commit = '7109f6a'
+# see DESCRIPTION to determine version,
```
Comment (verdurin) [inline easybuild/easyconfigs/d/DIALOGUE/DIALOGUE-1.0-20230228-foss-2021b-R-4.2.0.eb:5]: @boegel wonder whether it's worth clarifying here that you mean the file `DESCRIPTION` in the upstream code, not the `description` in the easyconfig, lest anyone make the dim mistake I did at first glance...

Comment (Micket) [inline easybuild/easyconfigs/d/DIALOGUE/DIALOGUE-1.0-20230228-foss-2021b-R-4.2.0.eb:5]:
```suggestion
# see DESCRIPTION file in source to determine version,
```

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-c1 - Linux Rocky Linux 8.6, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/dd7be82c350a3d7cc8f772fba9472d31 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**